ports/mimxrt/boards/MIMXRT1010_EVK/mpconfigboard.mk

```diff
@@ -3,6 +3,9 @@ MCU_VARIANT = MIMXRT1011DAE5A
 
 MICROPY_FLOAT_IMPL = single
 
+SRC_C = \
```

Comment:
Please use `+=` here so it doesn't override something else.

Reply:
Done.